components/GetStarted.tsx

```diff
@@ -24,8 +24,8 @@ export default function GetStarted() {
               Ready to Make an Impact?
             </h2>
             <p className="text-xl text-white/90 max-w-2xl mx-auto mb-12">
-              Join our community of contributors and help drive sustainable
-              technology forward through open source collaboration.
+              <b>Join our community of contributors and help drive sustainable
+              technology forward through open source collaboration.</b>
```

Comment:
Instead of making new html tags (<b>) you would actually use the tailwindclass "text-bold" to make this. Keeps the code more consistent i would say.